main.js

```diff
@@ -0,0 +1,64 @@
+// 1. Написати функцію, яка приймає 4 аргументи і шукає найменше число серед них
+const GET_MIN_NUM = function (num1, num2, num3, num4) {
+    const MIN = Math.min(num1, num2, num3, num4);
+}
+GET_MIN_NUM()
+
+//2. Написати функцію, яка приймає 2 аргументи та шукає найбільший спільний дільник для них. Якщо такого числа немає, то повідомляти про відсутність
+const GET_GREATEST_COMMON_DENOMINATOR = function (num1, num2) {
+    if ((typeof num1 !== 'number') || (typeof num2 !== 'number')) 
+      return false;
+      num1 = Math.abs(num1);
+      num2 = Math.abs(num2);
+    while(num2) {
+      const NUM3 = num2;
+      num2 = num1 % num2;
+      num1 = NUM3;
+    }
+    return num1;
+  }
+GET_GREATEST_COMMON_DENOMINATOR()
```

Comment:
Not all cases covered
What if
```suggestion
GET_GREATEST_COMMON_DENOMINATOR(100)
GET_GREATEST_COMMON_DENOMINATOR(-100, 20)
GET_GREATEST_COMMON_DENOMINATOR(''string data', 20)
GET_GREATEST_COMMON_DENOMINATOR(null, undefined)
```